CHANGELOG.md

```diff
@@ -43,6 +43,8 @@ release.
   ([#345](https://github.com/open-telemetry/semantic-conventions/pull/345))
 - Add messaging metrics
   ([#163](https://github.com/open-telemetry/semantic-conventions/pull/163))
+- Add remaining ECS fields to the user-agent namespace
```

Comment:
A release was cut in between, so please move this to the unreleased section of the markdown.